features/support/hooks.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable new-cap */
+import { OurWorld } from "../ultils/world";
+import {
+  After,
+  AfterAll,
+  Before,
+  BeforeAll,
+  setWorldConstructor,
+  Status,
+} from "@cucumber/cucumber";
+import fs from "fs";
+import path from "path";
+import os from "os";
+
+setWorldConstructor(OurWorld);
```

Comment:
Do we need `setWorldConstructor()` here?

Reply:
It's redundant. I will remove it.